frontend/tests/unit/service/permitService.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { permitService } from '@/services';
+import { appAxios } from '@/services/interceptors';
+
+import type { Permit } from '@/types';
+
+vi.mock('vue-router', () => ({
+  useRouter: () => ({
+    push: vi.fn(),
+    replace: vi.fn()
+  })
+}));
+
+const testPermit1: Permit = {
+  permitId: 'permitId',
+  permitTypeId: 0,
+  activityId: 'activityId',
+  issuedPermitId: 'issuedPermitId',
+  trackingId: 'trackingId',
+  authStatus: 'authStatus',
+  needed: 'needed',
+  status: 'status',
+  submittedDate: new Date().toISOString(),
+  adjudicationDate: new Date().toISOString()
+};
+
+const PATH = 'permit';
+const TEST_ID = 'testID';
+
+const getSpy = vi.fn();
+const putSpy = vi.fn();
+const deleteSpy = vi.fn();
+
+const updatePermitSpy = vi.spyOn(permitService, 'updatePermit');
+
+vi.mock('@/services/interceptors');
+vi.mocked(appAxios).mockReturnValue({
+  get: getSpy,
+  put: putSpy,
+  delete: deleteSpy
+} as any);
+
+beforeEach(() => {
+  vi.clearAllMocks();
+});
+
+describe('permitService test', () => {
+  it('calls creates a permit', async () => {
+    await permitService.createPermit(testPermit1);
+    expect(putSpy).toHaveBeenCalledWith(PATH, testPermit1);
```

Comment:
Better to always test `toHaveBeenCalledTimes` first, prior to `toHaveBeenCalledWith`. While the latter implies the former, a test failure report is more clear to the developer at a glance when they know the code isn't even being invoked at all.